gulp/sass.js

```diff
@@ -45,7 +45,9 @@ module.exports = (gulp, plugins, blueprint) => {
     ));
 
     blueprint.task("sass", "compile", ["icons", "sass-variables"], (project, isDevMode) => {
-        const sassCompiler = plugins.sass();
+        const sassCompiler = plugins.sass({
+            importer: require("node-sass-package-importer")({ cwd: project.cwd }),
```

Comment:
please put the `require` with all the other imports. no reason to do it inline here.